hostdetection/multi_thread_hd.py

```diff
@@ -55,6 +57,26 @@ def build_request(api_route, params):
     return urllib2.Request(api_route, data=data, headers=build_headers())
 # end of build_request
 
+def get_concurrency_limit(api_route):
+    '''
+    validate credentials and return API concurrency limit for the subscription
+    '''
+    try:
+        print "Calling %s " % api_route
+        req = build_request(api_route, {})
```

Comment:
@ganeshwaghmare can `call_api(api_route, params)` be used here?

Reply:
Done with the changes.